salt/orchestrate/aws/mitx_elb.sls

```diff
@@ -71,4 +73,64 @@ register_edx_{{ purpose_name }}_nodes_with_elb:
         {% endfor %}
     - require:
         - boto_elb: create_elb_for_edx_{{ purpose_name }}
+
+{% if edx_type == 'live' %}
+{% set elb_name = 'edx-studio-live-{env}'.format(
```

Comment:
Wouldn't it be better if this was templatized similar to the previous elb_name?

Reply:
Same as above